backend/db/refresh.py

```diff
@@ -97,7 +97,7 @@ def cli():
         '-l', '--use-local-db', action='store_true', default=False, required=False,
         help='Use local database instead of server.')
     parser.add_argument(
-        '-b', '--buffer-hours', default=48, required=False,
+        '-b', '--buffer-hours', default=0, required=False,  # we were defaulting to 48 for a while
```

Comment:
@Sigfried I removed the buffer period where we were fetching 48 hours past the timestamp of the last refresh, because: during the GitHub action, after it runs the refresh, it also runs the test you set up, and will throw an error if it notices anything missing.

Removing the buffer period will make sure that the action completes much more quickly if no data is there, and won't do any unnecessary writes to the database.